Implementations for git add, commit and remove

dextros2001 opened the pull request:
Also added Spock unit test class for the GitService

dextros2001 committed:
Implementations for git add, commit and remove

Tsolval reviewed (changes requested):
Page body shouldn't be the title of the file.  Maybe change 'addPageBody' to 'addFile' and provide file name and file contents as parameters.

dextros2001 committed:
Updated GitService to addPageBody method to addPage

This method now accepts parameters for pageTitle and pageBody. The
pageTitle parameter becomes the title of the page where the pageBody
param represents the page body content of the page.

dextros2001 commented:
Updated to address issues

Tsolval reviewed (approved):
Looks good, thanks! 👍
